nifi-nar-bundles/nifi-metrics-reporting-bundle/nifi-metrics-reporting-task/src/test/java/org/apache/nifi/metrics/reporting/task/MetricsReportingTaskTest.java

```diff
@@ -147,7 +147,8 @@ public void setUp() throws Exception {
         rootGroupStatus = new ProcessGroupStatus();
         innerGroupStatus = new ProcessGroupStatus();
         when(reporterServiceStub.createReporter(any())).thenReturn(reporterMock);
-        when(reporterServiceStub.getIdentifier()).thenReturn(REPORTER_SERVICE_IDENTIFIER);
+        // TODO java11 never invoked, Mockito 2.23.4 fails the test since it's not in lenient mode
```

Comment:
Probably should restore this until the Java 11 stuff goes in

Reply:
This turned out to be a change that needed to be made with the Mockito 2.x upgrade.  The mock interaction is established, but never used.  Mockito 2.28.2 by default has mocks operating in strict mode, and flags interactions that aren't used, so it was removed.  Since the PR for the Mockito 2.x upgrade is now in master and this PR has will be rebased against master, this shouldn't show up as a diff.